fedimint-core/src/db/mod.rs

```diff
@@ -797,7 +797,7 @@ pub trait IDatabaseTransactionOpsCore: MaybeSend {
     async fn raw_remove_entry(&mut self, key: &[u8]) -> Result<Option<Vec<u8>>>;
 
     /// Returns an stream of key-value pairs with keys that start with
-    /// `key_prefix`. No particular ordering is guaranteed.
+    /// `key_prefix`, sorted by key.
```

Comment:
Good catch, that was certainly a documentation bug (heck, there's even a version of this function that guarantees the reverse ordering :laughing:)